problem_builder/public/css/questionnaire.css

```diff
@@ -67,13 +67,13 @@
 .mentoring .questionnaire .feedback .close {
     cursor: pointer;
     position: absolute;
-    top: 5px;
-    right: 3px;
-    width: 18px;
-    height: 21px;
-    font-family: FontAwesome;
-    font-style: normal;
-    text-align: center;
+    top: 12px;
+    right: 12px;
+    font-size: 0;
```

Comment:
Unfortunately setting this to 0 makes the "close" button invisible in standard theme when `display: none` is disabled:
![noicon](https://user-images.githubusercontent.com/3189670/69930797-187a4300-14c5-11ea-91bf-de6496336b6f.png)

I also tested this with a custom theme and it seems to be working the same way.

Reply:
@Agrendalath I beg your pardon. I wrote SCSS instead CSS and that thing breaked into css file. I have fixed it. Please check now

Reply:
Ah, right. No worries!